### PR TITLE
build: update teams-js dependency

### PR DESCRIPTION
**Pull request by Yimin-Jin**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/27300222

### CODE REVIEW

File: templates/js/non-sso-tab/src/views/hello.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="/favicon.ico" />
     <link rel="stylesheet" type="text/css" href="/static/styles/custom.css" />
     <script
-      src="https://res.cdn.office.net/teams-js/2.17.0/js/MicrosoftTeams.min.js"
+      src="https://res.cdn.office.net/teams-js/2.19.0/js/MicrosoftTeams.min.js"
       integrity="sha384-xp55t/129OsN192JZYLP0rGhzjCF9aYtjY0LVtXvolkDrBe4Jchylp56NrUYJ4S2"
```
Comment (hund030) [inline templates/js/non-sso-tab/src/views/hello.html:9]: I think the integrity should be updated, too.

File: templates/js/non-sso-tab/src/views/hello.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="/favicon.ico" />
     <link rel="stylesheet" type="text/css" href="/static/styles/custom.css" />
     <script
-      src="https://res.cdn.office.net/teams-js/2.17.0/js/MicrosoftTeams.min.js"
+      src="https://res.cdn.office.net/teams-js/2.19.0/js/MicrosoftTeams.min.js"
```
Comment (hund030) [inline templates/js/non-sso-tab/src/views/hello.html:8]: What about using the latest version?

**Review by hund030 — COMMENTED** [review]
Please verify the templates work fine after the updates.

File: templates/js/non-sso-tab/src/views/hello.html (hunk 1 above)
Comment (Yimin-Jin) [reply templates/js/non-sso-tab/src/views/hello.html:9]: Sorry I forgot to update integrity

File: templates/js/non-sso-tab/src/views/hello.html (hunk 2 above)
Comment (Yimin-Jin) [reply templates/js/non-sso-tab/src/views/hello.html:8]: Ok, I'll try to update it to the latest version: 2.22.0 and update after verifying that it works.

**Review by hund030 — APPROVED** [review]

**Pull request merged**